pages/templates/pages/base_page.html

```diff
@@ -54,7 +54,9 @@
     <script src="{% static "dist/js/bundle.min.js" %}?v=12"></script>
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
     <meta name="google-site-verification" content="devyaHsbJD6w8nNEWsAu3SsODhEfijQqCOhFbuXdpCc" />
-    <link rel="icon" type="image/x-icon" href="{% static "dist/img/favicon.ico" %}" />
+    {% block favicon %}
+      <link rel="icon" type="image/x-icon" href="{% static "dist/img/favicon.ico" %}" />
```

Comment:
Maybe better to just put image url in block? Is anything else here ever going to change?

Reply:
Similar to the page title logic.